modules/programs/k9s.nix

```diff
@@ -86,14 +89,14 @@ in
         <https://k9scli.io/topics/aliases/> for supported values.
       '';
       example = literalExpression ''
-        alias = {
+        aliases = {
```

Comment:
in general literalExpression represents just the value so in this case it would be:

```
{
          # Use pp as an alias for Pod
          pp = "v1/pods";
        };
```

same remark for other examples